styles.css

```diff
@@ -193,6 +339,108 @@ ul {
   margin: 0 5px 0 5px;
 }
 
+.featured-stories-title {
+  color: #000;
+  font-size: 20px;
+}
+
+.featured-stories-text {
+  color: #333;
+  font-size: 12px;
+}
+
+/* Mobile */
+
+@media screen and (max-width: 420px) {
+  .ups {
+    left: 126px;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .section-1 {
+    margin-top: 50px;
+  }
+}
```

Comment:
- [ ] This should be within the other media query. If there is need to use this it means that the media queries are not properly set-up.